src/test/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketGitSCMRevisionTest.java

```diff
@@ -26,67 +26,52 @@
 import com.cloudbees.jenkins.plugins.bitbucket.api.BitbucketApi;
 import com.cloudbees.jenkins.plugins.bitbucket.client.BitbucketIntegrationClientFactory;
 import com.cloudbees.jenkins.plugins.bitbucket.endpoints.BitbucketCloudEndpoint;
-import edu.umd.cs.findbugs.annotations.NonNull;
-import hudson.Extension;
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.EnumSet;
-import java.util.List;
+import hudson.Util;
+import hudson.model.TaskListener;
 import jenkins.scm.api.SCMHead;
-import jenkins.scm.api.SCMHeadObserver;
-import jenkins.scm.api.SCMRevision;
-import jenkins.scm.api.mixin.ChangeRequestCheckoutStrategy;
-import jenkins.scm.api.trait.SCMHeadFilter;
-import jenkins.scm.api.trait.SCMSourceContext;
-import jenkins.scm.api.trait.SCMSourceRequest;
+import jenkins.scm.api.trait.SCMHeadAuthority;
 import jenkins.scm.api.trait.SCMSourceTrait;
-import jenkins.scm.api.trait.SCMSourceTraitDescriptor;
+
+import java.util.Arrays;
```

Comment:
I guess you forgot to run checkstyle 😅

Reply:
the build should fails, checkstyle is always run with normal maven. Let me check what's wrong

Reply:
Ya CI didn't run for some reason

Reply:
https://ci.jenkins.io/job/Plugins/job/bitbucket-branch-source-plugin/job/PR-152/11/git-2/ build 11 which is the latest atm, does not match the git commit from the latest commit here.

Reply:
The checkstyle was run in jenkins build and locally it work. The reason why it not fail is because [by default checkstyle](https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html#includeTestSourceDirectory) do not run against test classes. Would you enable also for test sources?

Reply:
Yes since the current checkstyle is minor and was already attempted to be applied for test classes 😅

Reply:
done